Bottom: fd6c1a26b6d4ef699e0fa745b0c808066f86ea40
Top:    9109aa686aa2ed444db6bb06c681b09d73781f6e
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 17:43:40 -0400

Refresh of mpc5200-dma-split

---

diff --git a/sound/soc/fsl/Makefile b/sound/soc/fsl/Makefile
index 56b2476..7731ef2 100644
--- a/sound/soc/fsl/Makefile
+++ b/sound/soc/fsl/Makefile
@@ -14,4 +14,3 @@ obj-$(CONFIG_SND_SOC_MPC8610) += snd-soc-fsl-ssi.o snd-soc-fsl-dma.o
 obj-$(CONFIG_SND_MPC52xx_DMA) += mpc5200_dma.o
 obj-$(CONFIG_SND_SOC_MPC5200_I2S) += mpc5200_psc_i2s.o
 
-
